templates/lidmaatschap.txt

```diff
@@ -0,0 +1,34 @@
+# E-mail dit volledig ingevuld naar: penningmeester@fronteers.nl 
+
+Naam:
+[VOOR-EN ACHTERNAAM]
+
+Adres: dient je persoonlijke adres te zijn
+[ADRES]
+
+E-mailadres:
+[EMAILADRES]
+
+Telefoonnummer:
+[TELEFOONNUMMER]
+
+Werkgever: we sturen de factuur niet naar je werkgever; je bent persoonlijk lid
+[WERKGEVER]
+
+Ja, ik ben student aan een hogere onderwijsinstelling: [JA/NEE]
+Om studentenkorting te ontvangen, dien je een bewijs van inschrijving van de hogere onderwijsinstelling waar je studeert aan het bestuur beschikbaar te maken. Stuur een scan mee!
```

Comment:
I think it would be a good idea to also add this description to the English version :)